winml/lib/Api.Ort/OnnxruntimeEngineBuilder.cpp

```diff
@@ -42,6 +42,11 @@ STDMETHODIMP OnnxruntimeEngineBuilder::CreateEngine(_winml::IEngine** out) {
                             ort_api);
   }
 
+  if (intra_op_num_threads_override_.has_value()) {
+    RETURN_HR_IF_NOT_OK_MSG(ort_api->SetInterOpNumThreads(session_options.get(), intra_op_num_threads_override_.value()),
+        ort_api);
```

Comment:
What's the ORT default if SetIntraOpNumThreads isn't called?

Reply:
I'll make a comment. But ORT default if SetIntraOpNumThreads isn't called is half the number of logical cores on the machine.